### PR TITLE
Render/utilize path property

### PR DESCRIPTION
**Pull request by ericcornelissen**
- [x] I've read the Contributing Guidelines and the Code of Conduct

# Description

Update simple-icons dependency to v1.10.0 (https://github.com/simple-icons/simple-icons/pull/1503) and utilize the new `path` property of exported icons in the rendering logic.

# Why is this change required?
n/a

# How did you test that?
The result should equal the original logic, so the original test suites should apply here as well.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "clean": "rimraf .rpt2_cache coverage dist *.log *.tgz"
   },
   "dependencies": {
-    "simple-icons": "^1.9.28",
+    "simple-icons": "^1.10.0",
```
Comment (sh7dm) [inline package.json]: should go separately, not related

File: src/VueSimpleIcon.ts [hunk 2]
```diff
@@ -11,26 +11,28 @@ import { renderError } from "./util";
...
       },
-      domProps: {
-        innerHTML: svgContent
-      }
-    });
+      [
+        createElement("title", title),
+        createElement("path", {
+          attrs: {
+            d: icon.path
```
Comment (sh7dm) [inline src/VueSimpleIcon.ts]: What's about multipath icons? have you tested them?

File: package.json (hunk 1 above)
Comment (sh7dm) [reply package.json]: oh, sorry, re-read description, that's required

File: src/VueSimpleIcon.ts (hunk 2 above)
Comment (ericcornelissen) [reply src/VueSimpleIcon.ts]: The whole idea of the effort was to make sure all icons have only a single path, for more background see: https://github.com/simple-icons/simple-icons/issues/1275 and https://github.com/simple-icons/simple-icons/pull/1277.

Hence all icons have only one path and it's found as a string (see https://github.com/simple-icons/simple-icons/blob/b5c346cdd9ff4fe51dd83357d258c61f609837b5/tests/icons.test.js#L25) of the path property of the icon.

**Pull request merged**